<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenrowsetWidget</class>
 <widget class="QWidget" name="OpenrowsetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>591</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>591</width>
    <height>618</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OPENROWSET</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection String</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineIP">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUsername">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Login:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePassword">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="driverBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SQLNCLI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SQLoleDb</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Driver:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>db:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connTestButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Test connection</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="openrowsetGroup">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>571</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>Query</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>551</width>
      <height>401</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="queryRun">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSelect">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSelect_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FROM</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineFrom">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rows:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="selectTOP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TOP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTOP">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="queryBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SELECT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XP_CMDSHELL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXEC</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="enableButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable openrowset &amp;&amp; xp_cmdshell</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRowsCount">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5000</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
